aepp/flowservice.py

```diff
@@ -477,8 +477,8 @@ def createFlow(
         source_connection_id: str = None,
         target_connection_id: str = None,
         schedule_start_time: str = None,
-        schedule_frequency: str = "minute",
-        schedule_interval: int = 15,
+        schedule_time_unit: str = "hour",
+        schedule_interval: int = 3,
```

Comment:
I wouldn't change the default values as this could break people who might be using this library already. 15 minutes is fine for sources.